cartridges/app_storefront_training/cartridge/client/default/js/main.js

```diff
@@ -0,0 +1,6 @@
+window.jQuery = window.$ = require('jquery');
+var processInclude = require('./util');
+
+$(document).ready(function () {
+    processInclude(require('./carousels/carousels'));
+});
```

Comment:
Always place a line break at the end of the file.

Reply:
Solved